docs/dashboards/filter-template-variables.md

```diff
@@ -203,3 +204,8 @@ To use a metrics variable in a query working in advanced query mode, do the fol
 
 1. Locate the name of the variable you'd like to use in the dropdown list and copy it.
 1. Go to your metrics query and insert *{{ var_name }}* in the query, substituting `var_name` with the actual name of the variable.
+
+
+## Limitation
```

Comment:
@JV0812 Couple of comments here. There's already a limitations section at the top of the page. Second the wording is quite right. You can use a multi-variable in the "selector" but only in the form of `<field>={{variable}}`. Adding `<field> matches "{{variable}}"` will only work after the first pipe.

Reply:
@oshoghi Updated, can you please check now.